### PR TITLE
Minor cleanup for slice::Chunks and ChunksMut

### PR DESCRIPTION
**Pull request by sdroege**
This only renames the `size` field to `chunk_size` in one of them for consistency, and changes an assertion to check for != 0 instead of > 0.

### CODE REVIEW

File: src/libcore/slice/mod.rs [hunk 1]
```diff
@@ -348,9 +348,9 @@ impl<T> SliceExt for [T] {
     }
 
     #[inline]
-    fn chunks(&self, size: usize) -> Chunks<T> {
+    fn chunks(&self, chunk_size: usize) -> Chunks<T> {
         assert!(size != 0);
```
Comment (frewsxcv) [inline src/libcore/slice/mod.rs]: `size` here needs to be updated

Comment (sdroege) [reply src/libcore/slice/mod.rs]: Indeed, thanks. Sorry for that!

**Comment by sdroege** [comment]
Also fixed another case of `size` in the documentation of both structs

**Review by dtolnay — APPROVED** [review]
Thanks!

**Pull request merged**